README.md

```diff
@@ -8,5 +8,22 @@ planner : MaxlPlanner (devloped at Mowito)
 Keyboard innput can be given at run time<br/> 
 robot will update path taking obstacle avoidance in account as goal is changed by keys i , l j to move it forword backword rigth left 
 
+# How to launch
+### System requirement
+Ubuntu 18/20<br/>
+ROS Melodic/Neotic (Ros1)<br/>
+Rviz (present by default if not [refer](http://wiki.ros.org/rviz/UserGuide))<br/>
+For further ROS package dependencies [refer](https://docs.google.com/document/d/1ax2jxiBoe7LoIBfpz3iLcvHPjz6RKT2QiCndPhsWyjA/edit)
+#### Repository requirement
+For running the simulation you must have Maxl Planner install which is mention above<br/>
```

Comment:
..........you must have the mw_maxl_planner repo installed. More information can be found on this link (add URL to the link)